File: abc162/b/b.py
def fizbuz(n):
    total = 0
    for i in range(n + 1):
        if i % 3 != 0 and i % 5 != 0:
            total += i
    return total


def main():
    n = int(input())
    print(fizbuz(n))


if __name__ == "__main__":
    main()

File: abc162/b/b.c
#include <stdio.h>

long long fizbuz(long long n) {
  long long total = 0;
  for (long long i = 0; i <= n; i++) {
    if (i % 3 != 0 && i % 5 != 0) {
      total += i;
    }
  }
  return total;
}

int main(void) {
  long long n;
  scanf("%lld", &n);
  printf("%lld\n", fizbuz(n));
  return 0;
}
